### PR TITLE
Add RSpec framework files

### PR DESCRIPTION
**Pull request by Fryguy**
@djberg96 Please review.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,17 @@
-language: node_js
-node_js:
+matrix:
```
Comment (Fryguy) [inline .travis.yml]: I'm not sure if this is the right way to do this...I basically need two completely different languages, and reading the docs, it seems the matrix is the way to do this.

Comment (Fryguy) [reply .travis.yml]: And I didn't want to do it the way the ui-classic does it because they nvm install on demand, and I was trying to find a way to use the node travis image directly.

**Comment by Fryguy** [comment]
cc @jrafanie @juliancheal I just wanted you to see what changes I had to do here to get multi-language support, and so if you have to blast out ruby upgrades, this repo is a little different.

File: .rspec [hunk 2]
```diff
@@ -0,0 +1,5 @@
+--require manageiq/spec/spec_helper
+--require spec_helper
```
Comment (djberg96) [inline .rspec]: Unless having a local spec_helper.rb is required, I would leave this line out.

File: spec/example_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+describe MigrationLogController do
+  it "stuff" do
+    expect(1).to eq 1
+  end
+end
```
Comment (djberg96) [inline spec/example_spec.rb]: Is this file required?

File: .rspec (hunk 2 above)
Comment (djberg96) [reply .rspec]: Oops, nm, I see that you added it.

**Comment by djberg96** [comment]
@Fryguy overall LGTM, though I'm not an expert on the JS stuff. @mturley or @vconzola Can either of you comment?

File: spec/example_spec.rb (hunk 3 above)
Comment (Fryguy) [reply spec/example_spec.rb]: It was my placeholder for your tests otherwise travis would have nothing to run.

Comment (djberg96) [reply spec/example_spec.rb]: Ah, ok, thanks.

File: .travis.yml [hunk 4]
```diff
@@ -1,7 +1,28 @@
...
-script: yarn test --runInBand
+dist: xenial
+matrix:
+  include:
+  - name: "Node.js: 8"
+    language: node_js
+    node_js:
+    - '8' # current stable
+    cache: yarn
+    script: yarn test --runInBand
```
Comment (Fryguy) [inline .travis.yml]: This part is basically the same as before, but added `cache: yarn` for performance.

Comment (mturley) [reply .travis.yml]: Sounds good to me.
